From: Jason Rhinelander <jason@imaginary.ca>
Date: Thu, 29 Oct 2015 12:16:20 -0400
Subject: Allow overriding MATHJAX_PATH

The given path is used as-is, even if it doesn't (yet?) exist.
---
 doc/CMakeLists.txt | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/doc/CMakeLists.txt b/doc/CMakeLists.txt
index d80e7fa..e80acc2 100644
--- a/doc/CMakeLists.txt
+++ b/doc/CMakeLists.txt
@@ -9,7 +9,9 @@ endif()
 # Can set NO_MATHJAX to true to skip looking for the system MathJax, in which
 # case the default (MathJax via public CDN) will be used by doxygen.
 set(mathjax_usable_path "")
-if (NOT NO_MATHJAX)
+if (MATHJAX_PATH)
+    set(mathjax_usable_path "${MATHJAX_PATH}")
+elseif (NOT NO_MATHJAX)
     find_package(MathJax 2.3)
     if(MATHJAX_FOUND)
         set(mathjax_usable_path "${MATHJAX_PATH}")
